_config.yml

```diff
@@ -17,10 +17,17 @@ exclude:
 collections:
   Releases:
     output: true
+  PackagesTest:
+    output: true
 
 defaults:
   - scope:
       path: ""
       type: Releases
     values:
       layout: releases
+  - scope:
+      path: ""
+      type: PackagesTest
```

Comment:
Just wondering, but: why are you calling the collection `PackagesTest` and not directly `Packages`?

Reply:
We weren't sure whether that would clash with the already existing `Packages/` folder since e.g. `semigroups.html` would then exist in `Packages/` and in `_Packages/`.